src/dev/build/tasks/patch_native_modules_task.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { deleteAll, download, untar } from '../lib';
+
+const BASE_URL = `https://storage.googleapis.com/native-modules`;
```

Comment:
We do :) https://github.com/elastic/node-re2/tree/pre-build currently builds to a temporary personal AWS bucket because of infra/legal issues, and the the files are copied to our GCP storage bucket, where we pull it from.

Reply:
Gotcha -- are we going to wait on merging this until those infra issues are resolved, or are we trusting the temporary personal bucket?

The SHA check we have would prevent a build from succeeding if the assets were tampered with though, right?

Reply:
I was planning on merging before the infra issues are resolved. I'd like to see whether or not https://github.com/uhop/node-re2 is amenable to using a pre-built version of `node-pre-gyp` so we don't have to maintain [our fork](https://github.com/elastic/node-re2) which does the one-off build. 

The SHA is generated after the build succeeds.

Reply:
I should have clarified, The SHA check we have would prevent **a Kibana** build from succeeding if the assets were tampered with though

Reply:
> I should have clarified, The SHA check we have would prevent a Kibana build from succeeding if the assets were tampered with though

Yup, absolutely! We just have to trust whomever/whatever built them and transferred them to that bucket (which was me and Travis CI).